pocketimdb/users/permission.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.permissions import BasePermission
+
+class UserAccessPermission(BasePermission):
+
+    message = 'You cannot access this resource.'
+    
+    def has_permission(self, request, view):
+        if str(request.user.id) == request.resolver_match.kwargs['user_pk']:
```

Comment:
ovaj user_pk mozes ovako izvuci `view.kwargs['user_pk']`

Reply:
Verovao ili ne prvo sam tako uradio i to sam sam izvalio, pa sam onda negde ovako nesto video i zamenio...

Reply:
Ispravljam to definitivno